src/modules/yield-farming/api.ts

```diff
@@ -1,8 +1,8 @@
-import { gql } from '@apollo/client';
 import BigNumber from 'bignumber.js';
```

Comment:
Would be good to keep logic pointing the graph. Keep the version of this file as one from development.